src/app/shared/tool.ts

```diff
@@ -34,6 +34,7 @@ export abstract class Tool implements OnInit, OnDestroy {
               private containerService: ContainerService,
               toolType: string) {
     this._toolType = toolType;
+    // this.isToolPublic = true;
```

Comment:
Take out commented code unless it is described well and would be useful in the future